### ISSUE TITLE
[BUG]: Paging within FAQs component is not working. Click on Down arrow icon and nothing happens

### ISSUE THREAD

**Issue by dbackstrong** [microsoft/powerapps-tools#161]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Interface(please complete the following information):**
 - Version date[e.g. 2020.04.03]

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mlopez1990** [comment]
Hi, i followed all steps for de configuration of this app, but i have a problem with the FAQS, usa can a solution for de run the module.

![error 1](https://user-images.githubusercontent.com/62559403/77339381-480dbf00-6cf1-11ea-9e54-cb735cb64321.PNG)

![ERROR 2](https://user-images.githubusercontent.com/62559403/77339466-6d9ac880-6cf1-11ea-9acd-f628d23159c9.PNG)

Thanks!

Regards

**Comment by mr-dang-msft** [comment]
@dbackstrong ,
Can you paste a screenshot of what you mean about the FAQ? Is the Answer field filled out in the SharePoint list?

**Comment by mr-dang-msft** [comment]
@mlopez1990 ,
Did you remove the existing connection to SharePoint and re-add your own?

**Comment by dbackstrong** [comment]
The FAQs seem to be corrected in the last version. They page as expected. Thx
________________________________

**Comment by dbackstrong** [comment]
After further testing, there remains an issue.  When clicking the FAQ menu link from an initial load of the Main App page, the paging does not work.  But, once I navigate to another option (i.e. Links) and return to FAQs, the paging works.  Paging for FAQs consistently does not work from the Home page on the initial load.
________________________________

**Comment by dbackstrong** [comment]
Bottom image, is the first time the FAQ page loads, when trying to page using the carrot at bottom of page, nothing happens.  But, the second time I try to access the page and all subsequent times, the paging works (see first image).[cid:0ee9113e-f16c-48c2-84a3-85c4439fdc73]Below is the first time accessing the FAQs page.
[cid:434e8fd2-abb0-4749-a802-cadadc862ced]
________________________________

**Comment by dbackstrong** [comment]
looking at the code, it seems the first time the FAQ page loads there is an action to collect user data and then a flag is set.  Doing this action must interfere with the building of the page and the paging feature.  Once the flag is set, it seems the page (and paging) work correctly.

**Comment by mr-dang-msft** [comment]
Hi @dbackstrong , by 'paging' are you talking about the navigation arrows that let you scroll through a gallery? If so, it has a known issue which is why we switched to scroll bars.

**Comment by dbackstrong** [comment]
Yes, it is the use of the arrow(s).  As stated, the arrows function properly the second time the page is accessed.  It is just the initial load when the arrow does not respond.  I do not see a scroll bar alternative on the page using Chrome or Edge.
________________________________
